### DISCUSSION TITLE
BVM

### DISCUSSION THREAD

**Discussion by k930503** [barreleye-labs/barreleye#20] [Q&A] [answered]
BVM을 1차적으로 어느 수준까지 지원하도록 구현해야할까?

**Comment by usiyoung** [comment, marked as answer]
> BVM을 1차적으로 어느 수준까지 지원하도록 구현해야할까?

솔리디티 실행되도록 이더리움 소스 참고해서 evm 그대로 카피해보자!

**Reply by k930503** [reply]
응응 내가 알아보고 있을게
